docs/reference/api/containers/box.rst

```diff
@@ -1,6 +1,8 @@
 Box
 ===
 
+.. include:: ../../api_status_key.rst
+
 .. rst-class:: widget-support
 .. csv-filter::
```

Comment:
Instead of including the same table on every page, how about adding a "What does this mean?" table caption that is a hyperlink to the main widget page, and a longer explanatory table on the widget-by-platform page. That would take up less space on the page, will result in less duplicated content, and gives us a chance to explain what we mean by "stable" and "beta". 

```suggestion
.. csv-filter:: :doc:`What does this mean? </reference/widgets_by_platform>`
```

with an extra CSS directive:
```
table.widget-support {
    caption-side: bottom;
}
```

Reply:
OK, I've actually put the caption above the table, so it can replace the "Availability" headers you recently added to Label and Button.